mappings/net/minecraft/entity/EntityContextImpl.mapping

```diff
@@ -3,12 +3,12 @@ CLASS net/minecraft/class_3727 net/minecraft/entity/EntityContextImpl
 		METHOD <init> (ZDLnet/minecraft/class_1792;)V
 			ARG 1 sneaking
```

Comment:
Arg on anonymous class...

Reply:
Will delete this one as there is no such constructor.